### PR TITLE
Remove reselect dev dep

### PR DESCRIPTION
**Pull request by upalatucci**
## 📝 Description

Remove reselect dev dep added in the past as a workaround for one sdk bug

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -153,7 +152,7 @@
     "tsconfig-paths-webpack-plugin": "^3.5.1",
     "typescript": "^4.5.5",
     "webpack": "^5.68.0",
-    "webpack-cli": "4.9.x",
+    "webpack-cli": "4.10.x",
```
Comment (upalatucci) [inline package.json:155]: After the upgrade i got a webpack error ```[webpack-cli] TypeError: cli.isMultipleCompiler is not a function```

Solved upgrading this

File: webpack.config.ts [hunk 2]
```diff
@@ -73,7 +73,8 @@ const config: Configuration = {
       },
       {
         test: /\.scss$/,
-        exclude: /node_modules\/(?!(@patternfly|@openshift-console\/plugin-shared)\/).*/,
+        exclude:
```
Comment (upalatucci) [inline webpack.config.ts:76]: now also `dynamic-plugin-sdk` has some scss files

**Review by upalatucci — COMMENTED** [review]
for reviewers

File: webpack.config.ts [hunk 3]
```diff
@@ -73,7 +73,8 @@ const config: Configuration = {
...
+          /node_modules\/(?!(@patternfly|@openshift-console\/plugin-shared|@openshift-console\/dynamic-plugin-sdk)\/).*/,
```
Comment (upalatucci) [inline webpack.config.ts:77]: now also `dynamic-plugin-sdk` has some scss files

**Review by avivtur — COMMENTED** [review]
Hi looks good :) 
we also have `@openshift-console/dynamic-plugin-sdk-webpack` version 0.0.8 and `@openshift-console/dynamic-plugin-sdk-internal` version 0.11 that we can upgrade to, can you please add them as well?

**Comment by upalatucci** [comment]
Sure

**Pull request merged**